common/client-core/Cargo.toml

```diff
@@ -115,6 +116,7 @@ features = ["wasm-bindgen"]
 
 [dev-dependencies]
 tempfile = { workspace = true }
+pretty_env_logger = { workspace = true}
```

Comment:
why is the env_logger a dev dependency? 
any logging backends should ideally be pulled from `bin-common` and I can't see why it would be dev specific

Reply:
Pinging @jmwample for this one

Reply:
I was unaware of `bin-common` and used the env logger while testing the stats collection to ensure that threads launched, channels sent, and reports logged as expected. The test is removed no because we have swapped to a network reporter, so I can probably remove this unless we want to write stats reports to logs in place of or along with network reporting.